lib/build.gradle

```diff
@@ -514,26 +513,31 @@ def setupJNILibraryBuildTasks(library, platform) {
 def createCopyNativeLibraryTask(library, targetPlatform) {
     if (CBL_NATIVE_LIBRARIES.contains(library)) return
 
-    CBL_NATIVE_LIBRARIES.add(library)
-    
-    def libs = ["macos/x86_64/libLiteCore.dylib", 
-                "linux/x86_64/libLiteCore.so", 
-                "windows/x86_64/LiteCore.dll"]
-    for (lib in libs) {
-        def libFile = file("${CBL_CORE_NATIVE_DIR}/${lib}")
-        if (libFile.exists()) {
-            CBL_NATIVE_LIBRARIES.add(libFile)
+    def libPath = getNativeLibraryResourcePath(targetPlatform)
+    task copyJni(type: Copy, dependsOn: LiteCoreJNISharedLibrary) {
```

Comment:
Not a complaint, but I don't see the task LiteCoreJNISharedLibrary.  Where is it???